### PR TITLE
fixtures: Adds fixtures for files

### LINKED ISSUE #2: fixtures: files-rest initialisation
Add support for initializing storage directory and Files-REST default location.

[Example fixture](https://github.com/zenodo/zenodo/blob/master/tests/unit/conftest.py#L315-L341)

Perhaps add possibility to parameterize it to create multiple locations.

Helper methods for easy creation of buckets and object versions.

### PR DESCRIPTION
**Pull request by pazembrz**
Adds fixture for creating default Location,
Adds fixture for creating Bucket from direcotry with files.

closes #2

### CODE REVIEW

File: pytest_invenio/fixtures.py [hunk 1]
```diff
@@ -554,3 +554,38 @@ def _get_screenshots_dir():
...
+
+
+@pytest.fixture(scope="function")
+def bucket_from_dir(db, location):
+    """Creates bucket from provided directory"""
+    def create_bucket_from_dir(source_dir, location_obj=None):
+        if not location_obj:
+            from invenio_files_rest.models import Bucket, Location, \
+                ObjectVersion
+            location_obj = Location.get_default() or location
```
Comment (Glignos) [inline pytest_invenio/fixtures.py]: Is ```location``` a defined variable here?

Comment (Glignos) [reply pytest_invenio/fixtures.py]: Ah never mind that!

File: pytest_invenio/fixtures.py [hunk 2]
```diff
@@ -554,3 +554,38 @@ def _get_screenshots_dir():
...
+    db.session.add(location_obj)
+    db.session.commit()
+
+    yield location_obj
+
+    shutil.rmtree(location_obj.uri)
+
+
+@pytest.fixture(scope="function")
+def bucket_from_dir(db, location):
```
Comment (Glignos) [inline pytest_invenio/fixtures.py]: Since location includes the db fixture you don't need to add it separately. :)

Comment (pazembrz) [reply pytest_invenio/fixtures.py]: Yes, but I'm using db inside this fixture, and it's easier(and nicer) to pass it as a db fixture than import form other modules to this fixture.

**Pull request merged**